gallery/src/data-visualization/ReactVisBarGraphPage.tsx

```diff
@@ -0,0 +1,141 @@
+          Data points for horizontal bar graphs utilize the <code className="nx-code">HorizontalBarSeriesPoint</code>
+          {' '} interface.
+        </p>
+
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>x</NxTableCell>
+              <NxTableCell>VerticalBarSeries: string/number <br /> HorizontalBarSeries: number</NxTableCell>
+              <NxTableCell>yes</NxTableCell>
+              <NxTableCell>
+                The x position in coordinates of the box to be used.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>y</NxTableCell>
+              <NxTableCell>VerticalBarSeries: number <br /> HorizontalBarSeries: string/number</NxTableCell>
+              <NxTableCell>yes</NxTableCell>
+              <NxTableCell>
+                The y position in coordinates of the box to be used.
+              </NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      </section>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Basic API Reference</h3>
+        </header>
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>barWidth</NxTableCell>
+              <NxTableCell>number/string/object</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                The percentage for which each bar fills the designated bucket. 1.0 means that the bar fills the whole
+                bucket (no padding between bars), while a smaller percentage means more whitespace between the bars.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>color</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                Assigns the specified color to the series.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>stroke</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                In LineSeries, stroke is used instead of color to specify the color of the lines.
```

Comment:
But this page isn't about `LineSeries`

Reply:
> But this page isn't about `LineSeries`

Should have been removed from this page, so has been removed.